source/ProtonPack/ProtonPack.ino

```diff
@@ -549,6 +549,13 @@ void loop() {
   }
 }
 
+void debug(String message) {
```

Comment:
Maybe encase the entire debug function around the DEBUG_SEND_TO_CONSOLE to cut down on the memory footprint? Though the compiler may be removing it from the code if it's never called, but just to be safe perhaps....

Reply:
I'll back this out entirely. I got what I needed from the messages, and 99.9% of the time they're not needed. If we need debugs I'll just stick to adding them when needed.